integration/spark/app/src/test/java/io/openlineage/spark/agent/InvalidTablesInJdbcInputsTest.java

```diff
@@ -0,0 +1,261 @@
+/*
+/* Copyright 2018-2024 contributors to the OpenLineage project
+/* SPDX-License-Identifier: Apache-2.0
+*/
+
+package io.openlineage.spark.agent;
+
+import static io.openlineage.spark.agent.SparkTestUtils.createHttpServer;
+import static io.openlineage.spark.agent.SparkTestUtils.createSparkSession;
+import static org.assertj.core.api.Assertions.assertThat;
+
+import com.sun.net.httpserver.HttpServer;
+import io.openlineage.client.OpenLineage;
+import io.openlineage.spark.agent.SparkTestUtils.OpenLineageEndpointHandler;
+import io.openlineage.spark.agent.SparkTestUtils.PostgreSQLTestContainer;
+import io.openlineage.spark.agent.SparkTestUtils.SchemaRecord;
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.nio.charset.StandardCharsets;
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+import lombok.extern.slf4j.Slf4j;
+import org.apache.spark.sql.SparkSession;
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.Test;
+import org.testcontainers.containers.PostgreSQLContainer;
+import org.testcontainers.junit.jupiter.Testcontainers;
+import org.testcontainers.utility.DockerImageName;
+
+@Tag("integration-test")
+@Testcontainers
+@Slf4j
+class InvalidTablesInJdbcInputsTest {
```

Comment:
Does this test really verify invalid tables?

Reply:
++ not good name for test :)

Reply:
fixed